LanguageExt.Core/LanguageExt.Core.csproj

```diff
@@ -56,4 +70,87 @@
+    </Compile>
+    <Compile Update="HKT\SyncAsync\HKT.SyncAsync.Extensions.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>HKT.SyncAsync.Extensions.tt</DependentUpon>
+    </Compile>
+    <Compile Update="HKT\Sync\HKT.Extensions.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>HKT.Extensions.tt</DependentUpon>
+    </Compile>
+    <Compile Update="HKT\Sync\HKT.Prelude.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>HKT.Prelude.tt</DependentUpon>
+    </Compile>
+    <None Update="HKT\AsyncSync\HKT.AsyncSync.Extensions.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>HKT.AsyncSync.Extensions.tt</DependentUpon>
+    </None>
+    <None Update="HKT\Async\HKT.Async.Extensions.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>HKT.Async.Extensions.tt</DependentUpon>
+    </None>
+    <None Update="HKT\SyncAsync\HKT.Async.Extensions.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+    </None>
+    <None Update="HKT\SyncAsync\HKT.Async.Extensions.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>HKT.Async.Extensions.cs</LastGenOutput>
+    </None>
+    <None Update="HKT\SyncAsync\HKT.SyncAsync.Extensions.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>HKT.SyncAsync.Extensions.tt</DependentUpon>
+    </None>
+    <None Update="HKT\SyncAsync\HKT.SyncAsync.Extensions.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>HKT.SyncAsync.Extensions.cs</LastGenOutput>
+    </None>
+    <None Update="HKT\Sync\HKT.Extensions.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>HKT.Extensions.tt</DependentUpon>
+    </None>
+    <None Update="HKT\Sync\HKT.Prelude.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>HKT.Prelude.tt</DependentUpon>
+    </None>
+    <None Update="HKT\Sync\HKT.Prelude.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>HKT.Prelude.cs</LastGenOutput>
+    </None>
+    <None Update="HKT\Sync\HKT.Extensions.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>HKT.Extensions.cs</LastGenOutput>
+    </None>
+    <None Update="HKT\Async\HKT.Async.Extensions.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>HKT.Async.Extensions.cs</LastGenOutput>
+    </None>
+    <None Update="HKT\AsyncSync\HKT.AsyncSync.Extensions.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>HKT.AsyncSync.Extensions.cs</LastGenOutput>
+    </None>
+  </ItemGroup>
+  <ItemGroup>
+    <Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />
+  </ItemGroup>
 </Project>
```

Comment:
You'll need to revert all changes to this file, this shouldn't need to be updated.  Presumably it was a merge issue of some sort?